packages/core/README.md

```
@@ -1,19 +1,40 @@
+The `@gardenfi/core` package provides an abstraction layer over the `@catalogfi/wallets` (which contains different types of Bitcoin and EVM wallets) and `@gardenfi/orderbook` packages. It offers a simple interface for performing atomic swaps.
+
+## Table of Contents
+
+1. [Installation](#installation)
+2. [Setup](#setup)
+   1. [Node](#node)
+   2. [Browser](#browser)
+      1. [Vite](#vite)
+      2. [Webpack](#webpack)
+3. [Documentation](#documentation)
+4. [Contributions](#contributions)
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)